loops/loop17.go

```diff
@@ -0,0 +1,57 @@
+// Is the number divisible by 7
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	// This code is from loop15.go
+	var count, n int =  0, 534
+	temp := n
+	digits := make([]int, 0, 1)
+	divisible := make([]int, 0, 1)
+	for temp != 0 {
+		digits = append(digits, temp % 10)
+		temp = temp / 10
+	}
+	for i:=2 ; i <= n; i++ {
+		if i == 2 {
+			if ifExists(digits[0], []int{0,2,4,6,8}) == true {
```

Comment:
Don't think an explicit "== true" is needed here.

Reply:
Pushed the latest code.